mlflow/utils/databricks_utils.py

```diff
@@ -738,54 +738,107 @@ def _init_databricks_cli_config_provider(entry_point):
     """
     notebook_utils = entry_point.getDbutils().notebook()
 
-    class DynamicConfigProvider(DatabricksConfigProvider):
-        def get_config(self):
-            logger = entry_point.getLogger()
-            try:
-                from dbruntime.databricks_repl_context import get_context
-
-                ctx = get_context()
-                if ctx and ctx.apiUrl and ctx.apiToken:
-                    return DatabricksConfig.from_token(
-                        host=ctx.apiUrl, token=ctx.apiToken, insecure=ctx.sslTrustAll
+    dbr_version = os.environ["DATABRICKS_RUNTIME_VERSION"]
+    dbr_version_splits = dbr_version.split(".", maxsplit=2)
+    dbr_major_minor_version = (int(dbr_version_splits[0]), int(dbr_version_splits[1]))
+
+    if dbr_major_minor_version >= (13, 2):
```

Comment:
Do we need a guard here if the dbr_major_minor_version doesn't resolve to a tuple of int?

Reply:
what's the case of not a tuple of int ? 
and how to handle it ?

Reply:
Right now we'll get a really confusing error message if dbr_version doesn't fetch. Should we raise if it doesn't return a splittable string?

Reply:
OK. sounds good!